### ISSUE TITLE
Toast History access failure on PowerShell 7.2.1

### ISSUE THREAD

**Issue by IanBAdams** [Windos/BurntToast#190]
### Steps to Reproduce

```powershell
Get-BTHistory -AppId 'Windows.Defender.SecurityCenter'

Note: I've tried the code with a number of different PowerShell versions. The code also fails with PowerShell 7.1.5 but works successfully with Windows PowerShell 5.1 and PowerShell 7.0.8.

Note2: I am aware of the issue 'Toast History access and Toast removal failures on PowerShell 7.1 #136'. However I believe this is a different issue because: 
- The actual behaviour is different for the two issues.
- I am successfully using Remove-BTNotification within my scripts (and was able to do so with PowerShell 7.1).
```

### Expected Behavior

The Get-BTHistory function to return the toast notifications in the Action Center. In my case should display something like:

ExpirationTime        : 02/02/2022 03:38:44 +00:00
Content               : Windows.Data.Xml.Dom.XmlDocument
Tag                   : YWD0SOEHhUSFvy/d.0
SuppressPopup         : False
Group                 : 23
RemoteId              :
NotificationMirroring : 1025960913
Priority              : Default
Data                  :
ExpiresOnReboot       : False

### Actual Behavior

Returns:

ObjRef                     : WinRT.ObjectReferenceWithContext`1[WinRT.IInspectable+Vftbl]
ThisPtr                    : 1803460262104
NativeObject               : WinRT.ObjectReferenceWithContext`1[WinRT.IInspectable+Vftbl]
HasUnwrappableNativeObject : True
AdditionalTypeData         : {[System.RuntimeTypeHandle, ABI.System.Collections.IEnumerable+AdaptiveFromAbiHelper]}

### Experience with Toast

Intermediate

### Environment data: PowerShell

```text
Name                           Value
----                           -----
PSVersion                      7.2.1
PSEdition                      Core
GitCommitId                    7.2.1
OS                             Microsoft Windows 10.0.19044
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

### Environment data: OS

```text
OsName         : Microsoft Windows 10 Pro
OsVersion      : 10.0.19044
OsArchitecture : 64-bit
```

### Environment data: BurntToast

```text
ModuleType Version    PreRelease Name                                PSEdition ExportedCommands
---------- -------    ---------- ----                                --------- ----------------
Script     0.8.5                 BurntToast                          Desk      {Get-BTHistory, New-BTAction, New-BTAppId, New-BTAudio…}
```

### Consent to Attribution

- [X] Please shout me out if I've inspired a blog post!

**Comment by Windos** [comment]
Yeah, this is the same issue as #136, there were some underlying .NET changes in PowerShell 7.1 which broke a lot of stuff and evidently, I missed applying the fix to `Get-BTHistory`.

This'll be sorted in v1.

**Comment by IanBAdams** [comment]
Many thanks for getting back to me on this.

At the moment I'm running `Get-BTHistory` within a Windows PowerShell session (5.1) which is called from PowerShell 7.2.1 so I'm in no hurry for a fix for this.

I'll keep an eye on v1 and once you add `Get-BTHistory` (or it's replacement) I'll test / amend my scripts as necessary. I'm only using a fraction of the capabilities of Burnt Toast (use the commands `New-BTHeader, New-BurntToastNotification, Remove-BTNotification and Get-BTHistory.`) so shouldn't take me long to make the changes required by v1.

**Comment by Windos** [comment]
Hello, and thank you for your contribution to BurntToast.

Today marks the official release of [BurntToast v1.0.0](https://github.com/Windos/BurntToast/releases/tag/v1.0.0). As part of this major milestone and to ensure the project starts its next chapter with a clean slate, I am declaring "issue bankruptcy."

This decision is explained in more detail in the official announcement post: [Ten Years Toasting: BurntToast Hits the Big v1!](https://toastit.dev/2025/07/22/burnttoast-v1/)

As a result, I am closing all outstanding issues and pull requests, including this one. Please do not interpret this as a lack of interest in the problem or suggestion you've raised. This is a necessary step to clear a very old backlog and focus development efforts on the current state of the module.

**If this topic is still relevant to you:**

1.  Please update to the latest version of BurntToast (`v1.0.0` or newer).
2.  Verify that your issue still exists in the new version.
3.  If it does, please **open a new issue**. You are welcome to link back to this one for historical context.

I hope this reset will allow me to be more responsive to community feedback going forward.

Thank you for your understanding and for being part of the BurntToast journey.
